components/rmrk/Gallery/History.vue

```diff
@@ -245,10 +283,9 @@ export default class History extends mixins(ChainMixin, KeyboardEventsMixin) {
     for (const newEvent of this.events) {
```

Comment:
this is far from refactored tbh

Reply:
better just I would move it to the utils and replace `for` loop with `Array.map`